build.cake

```diff
@@ -5,14 +5,22 @@
 // ARGUMENTS
 //////////////////////////////////////////////////////////////////////
 
+const string DefaultVersion = "1.0.0.0";
```

Comment:
Why the random const?

Reply:
Because it's the default and I compare to it below so to prevent typos.

Reply:
You can achive the same with the argument.